verticapy/performance/vertica/qprof.py

```diff
@@ -2139,6 +2138,71 @@ def get_qsteps(
         return vdf
 
     # Step 5: Query plan
+    def get_qplan_tr_order(
+        self,
+    ) -> list[int]:
+        """
+        Returns the Query Plan Temp
+        Relations Order.
+        It is used to sort correctly
+        the temporary relations in
+        the final Tree.
+
+        Returns
+        -------
+        list
+            List of the different
+            temp relations index.
+
+        Examples
+        --------
+        First, let's import the
+        :py:class:`~verticapy.performance.vertica.qprof.QueryProfiler`
+        object.
+
+        .. code-block:: python
+
+            from verticapy.performance.vertica import QueryProfiler
+
+        Then we can create a query:
+
+        .. code-block:: python
+
+            qprof = QueryProfiler(
+                "select transaction_id, statement_id, request, request_duration"
+                " from query_requests where start_timestamp > now() - interval'1 hour'"
+                " order by request_duration desc limit 10;"
+            )
+
+        We can easily call the function to get
+        the final order:
+
+            .. ipython:: python
+
+                qprof._get_qplan_tr_order()
+
+        .. note::
+
+            For more details, please look at
+            :py:class:`~verticapy.performance.vertica.qprof.QueryProfiler`.
+        """
+        query = f"""
+            SELECT 
+                REGEXP_SUBSTR(step_label, '\\d+')::INT
+            FROM v_internal.dc_plan_steps
+            WHERE 
+                transaction_id={self.transaction_id}
+            AND statement_id={self.statement_id}
+            AND step_label ILIKE '%TempRelWrite%' 
+            ORDER BY step_key ASC;"""
+        query = self._replace_schema_in_query(query)
+        res = _executeSQL(
+            query,
+            title="Getting the corresponding query",
+            method="fetchall",
+        )
+        return list(dict.fromkeys([q[0] for q in res]))
+
```

Comment:
What will happen with more complicated TempRel queries, will their plan trees be displayed horizontally all the way to the right? Is there a limit on how wide the horizontal display is?

Reply:
yes, it is displayed horizontally. There is no limit.